pyccel/stdlib/math/pyc_math.f90

```diff
@@ -0,0 +1,149 @@
+module pyc_math
```

Comment:
All the functions in this file should be declared as `pure`, and maybe even as `elemental`.

Reply:
yes, using the elemental functions will let us use this function for NumPy too.
but I remember that @saidctb said we will recreate this library in python and use it as a header in the future.

Reply:
I agree with @rakati . Pure, yes. But `elemental` does not reflect the behaviour in python

Reply:
Adding `pure` is enough.